.gitpod.yml

```diff
@@ -2,7 +2,7 @@ image: openjdk:8
 vscode:
   extensions:
     - scala-lang.scala@0.3.9:kklqw+c/dNRmtTU8B5repw==
-    - scalameta.metals@1.9.0:PtE/gdeE1YPL6It4tEfqpg==
+    - scalameta.metals@1.9.6:r4TvVGu3A2++o0cbEXt2Ig==
```

Comment:
Should be fine, I wonder how does the integration with openvsx can handle it. I would be cool to have something like `scalameta.metals@latest`, though I think openvsx might work only per user.

Reply:
Yea I have no idea how to actually get this hash. I had to start it, uninstall it, and then install it again to get the latest, and then the has was updated and I just grabbed it. I'm assuming there is an easier way than that to do it.